frontend/src/components/forms/Signup.tsx

```diff
@@ -72,72 +72,59 @@ const Signup = ({
     return <Redirect to={HOME_PAGE} />;
   }
 
-  if (toggle) {
+  if (toggle) {   
     return (
       <Flex h="100vh">
         <Box w="47%">
           <Flex
-            display="flex"
-            alignItems="flex-start"
-            position="absolute"
-            top="17.5%"
-            left="6%"
-            w="100%"
-          >
-            <Text variant="login" position="absolute">
-              Sign Up
-            </Text>
-          </Flex>
-          <Flex
-            h="40%"
-            w="36%"
-            top="28%"
-            left="6%"
+            h="70%"
+            marginTop="15%"
             direction="column"
-            position="absolute"
+            justify="center"
             justifyContent="space-between"
+            align="center"
           >
-            <Box>
+            <Box mr="58%">
```

Comment:
To get the left align you want you can just use `textAlign` instead:

```suggestion
            <Box w="80%" textAlign="left">
```